Added OMNY + Nebula

ausernameisnotavailable opened the pull request:
**Type of pull request:** product addition

**Related issues:** closes #142.

ausernameisnotavailable committed:
Add files via upload

ausernameisnotavailable committed:
Create omny.toml

ausernameisnotavailable committed:
Update omny.toml

ausernameisnotavailable committed:
Fix formatting issues in omny.toml

ausernameisnotavailable committed:
Added ausernameisnotavailable

ausernameisnotavailable committed:
Added Nebula logo

ausernameisnotavailable committed:
Create nebula.toml

doamatto commented on products/nebula.toml:
```diff
@@ -0,0 +1,60 @@
+
+[rubric.behavioral-marketing]
+value = "no"
+citations = ["and no ads in sight."]
```
This isn't a quote that shows that it does not do behavioural marketing. 

> We will not use any kind of technology that utilizes your microphone or camera to gather data about you for marketing purposes. 

This is a somewhat close quote, maybe the second bullet point on the privacy policy is a better bet.

doamatto commented on products/nebula.toml:
```diff
@@ -0,0 +1,60 @@
+description = "Nebula is a creator-owned video streaming platform"
+slug = "nebula"
+hostnames = [ "nebula.tv" ]
+sources = [ "https://nebula.tv/privacy", "https://nebula.tv" ]
```
<https://nebula.tv> isn't a privacy policy, it's just the homepage; unless I'm missing something?

doamatto commented on products/omny.toml:
```diff
@@ -0,0 +1,62 @@
+description = "OMNY is a public transit payment system in the New York City area"
+slug = "omny"
+hostnames = [ "omny.info" ]
+sources = [ "https://omny.info/privacy-policy/420" ]
```
Seems this just redirects to <https://omny.info/privacy-policy/>; might as well use that URL instead.

doamatto commented on products/omny.toml:
```diff
@@ -0,0 +1,62 @@
+
+[rubric.behavioral-marketing]
+value = "no" 
+notes = ["No mentioning of any behavioural marketing could be found. Users may recieve non-personalised emails, but may opt-out."]
```
Some things point to this not being true:

> We use information that we collect or process, including Personal Information, Online Activity Information, and Anonymous Information, as permitted under applicable law and consistent with this OMNY Privacy Policy and the OMNY Terms of Service. More specifically, we use the information we collect for the following purposes: 
> ...
> to perform analytics, quality control, market research, and to determine the effectiveness of our OMNY Services, promotional campaigns, to improve our current products, OMNY Services, and develop new products and services; 
> ...
> In addition to the uses specifically identified above, there may be instances where you request information and OMNY Services from us that are not described in this OMNY Privacy Policy.

doamatto commented on products/omny.toml:
```diff
@@ -0,0 +1,62 @@
+[rubric.data-collection-reasoning]
+value = "mostly"
+citations = ["We use information that we collect or process, including Personal Information, Online Activity Information, and Anonymous Information, as permitted under applicable law and consistent with this OMNY Privacy Policy and the OMNY Terms of Service. More specifically, we use the information we collect for the following purposes:"]
+notes = ["They list what the data is used for, although there is no reason that they should need to be collecting 'Online Activity Information'"]
```
"Online Activity Information" is defined in 2.2

doamatto commented on products/omny.toml:
```diff
@@ -0,0 +1,62 @@
+
+[rubric.law-enforcement]
+value = "always"
+citations = ["We may be required to share your Personal Information pursuant to a court order or other legal or regulatory obligation or process."]
```
`always` may not be the best fit here. `reasonable` or `strict` seem to be better suited, since they say they need "a court order or other legal or regulatory obligation"

doamatto commented on products/omny.toml:
```diff
@@ -0,0 +1,62 @@
+
+[rubric.third-party-collection]
+value = "no" 
+citations = ["If we collect other information about you using a method that is not specifically listed here, then we will use it in accordance with this OMNY Privacy Policy."]
```
@milesmcc This really brings the question that there's no "unspecified" for situations like these where the policy is ambiguous in its uses; I'll make an issue about it so that it doesn't block the progress of this PR, but it seems like something work discussing.

doamatto replied:
See #148.

doamatto reviewed (changes requested):
Here's what I've got so far; I'll look at it again once changes are made.

ausernameisnotavailable committed:
Update nebula.toml

ausernameisnotavailable committed:
Update omny.toml

doamatto commented on products/nebula.toml:
```diff
@@ -0,0 +1,59 @@
+[rubric.noncritical-purposes]
+value = "na" 
+citations = ["We will not collect any more information than needed to provide you with a great experience."]
+notes = ["Data is only collected to provide what is described as a great experience. It is unclear what this means."]
```
If it's unclear if non-critical data is collected, why is this `na`? It may make more sense to use `no`.

doamatto reviewed (changes requested):
Here's some more changes for `products/nebula.toml`. I'll check the OMNY file later today.

doamatto commented on products/omny.toml:
```diff
@@ -0,0 +1,62 @@
+name = "OMNY"
+description = "OMNY is a public transit payment system in the New York City area"
```
Fix the check failure below.

> Product 'OMNY' must have a valid description that is a full sentence ends with a period

doamatto commented on products/omny.toml:
```diff
@@ -0,0 +1,62 @@
+[rubric.data-breaches]
+value = "no"
+citations = ["By using the OMNY Services, you acknowledge and agree that there is a risk that unauthorized third parties may engage in illegal activity, such as hacking or intercepting transmissions of Personal Information. You agree that the MTA is not responsible for any data or Personal Information obtained"]
+notes = ["OMNY does not accept responsibility in the event of a data breach, meaning they are unlikely to notify their users."]
```
The speculation in the note isn't necessary. You could probably omit the note altogether: the citation is sufficient, in my opinion.

doamatto commented on products/omny.toml:
```diff
@@ -0,0 +1,62 @@
+[rubric.security]
+value = "somewhat"
+citations = ["We have implemented physical, technical, and administrative security measures and controls designed to prevent unauthorized access to, or disclosure of, Personal Information that you provide to us. Additionally, we require our third party providers to implement security measures and to otherwise handle Personal Information consistent with this OMNY Privacy Policy and applicable law. ...
+notes = ["Very vague."]
```
Note is unnecessary.

doamatto commented on products/omny.toml:
```diff
@@ -0,0 +1,62 @@
+[rubric.revision-notify]
+value = "no"
+citations = ["We may update this OMNY Privacy Policy from time to time. If we make changes to this OMNY Privacy Policy, then we will post them on the Website and indicate the Effective Date. Your continued use of the OMNY Services constitutes your agreement with this OMNY Privacy Policy and any updates to it."]
+notes = ["You would need to check the website to find if the privacy policy had been modified."]
```
No need to define the criteria of `revision-notify` and what the value stands for; this is on the product page.

doamatto reviewed (changes requested):
Here's a few more requested changes for you. I'll check back once I see the tests passing or if you let me know sooner.

ausernameisnotavailable committed:
Update nebula.toml

ausernameisnotavailable committed:
Update omny.toml

ausernameisnotavailable committed:
Update nebula.toml

doamatto committed:
Merge branch 'master' into master

doamatto committed:
fix(product): changes to OMNY

- Fix TOML formatting
- Improve line formatting
- Fix score for list-collected

doamatto committed:
fix(product): changes to Nebula

- Updated score and citation behavioral-marketing (no -> yes-opt-out)
- Updated citation for data-breaches
- Updated data-collection-reasoning (no -> mostly)
- Updated citation for data-deletion
- Updated score and citation for history (no -> last-modified)
- Updated score and citation for law-enforcement (strict -> reasonable)
- Updated score and citation for list-collected (no -> generally)
- Updated score and citation for noncritical-purposes (no -> opt-out-some)
- Updated citation for revision-notify
- Updated score and citation for security (no -> somewhat)
- Updated score and citation for third-party-access (yes-unspecified-critical -> yes-unspecified)
- Updated score and citation for third-party-collection (no -> yes)
